### ISSUE TITLE
org.junit.jupiter.api does not exist

### ISSUE THREAD

**Issue by fizruk** [objectionary/eoc#10]
Trying out `eoc` on macOS 12.4. Getting these errors:

```
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /.../.eoc/generated-sources/EOorg/EOeolang/EOtestTest.java:[13,29] package org.junit.jupiter.api does not exist
[ERROR] /.../.eoc/generated-sources/EOorg/EOeolang/EOtestTest.java:[14,29] package org.junit.jupiter.api does not exist
```

Apparently, this should be added in dependencies or installed by the user, but I did not find relevant information in the README.

UPDATE: This only happens when there is `+junit` directive:

```
+package org.eolang
+junit

...
```

**Comment by yegor256** [comment]
@rultor release, tag is `0.2.3`

**Comment by yegor256** [comment]
@fizruk I believe it's fixed, try version 0.2.3
